src/profile-logic/marker-styles.js

```diff
@@ -121,30 +127,20 @@ export const markerStyles: MarkerStyles = {
     borderRight: colors.RED_50,
     squareCorners: true,
   },
-  GCSlice: {
-    ...defaultStyle,
-    background: colors.ORANGE_50,
-  },
-  GCMinor: {
-    ...defaultStyle,
-    background: colors.ORANGE_50,
-  },
-  ForgetSkippable: {
-    ...defaultStyle,
-    background: colors.ORANGE_50,
-  },
-  IdleForgetSkippable: {
-    ...defaultStyle,
-    background: colors.ORANGE_50,
-  },
-  CCSlice: {
-    ...defaultStyle,
-    background: colors.ORANGE_50,
-  },
-  IdleCCSlice: {
+  // Memory:
+  GCMajor: {
     ...defaultStyle,
+    squareCorners: true,
     background: colors.ORANGE_50,
   },
+  GCSlice: memoryStyle,
+  GCMinor: memoryStyle,
+  CCSlice: memoryStyle,
+  /** CC */ IdleCCSlice: memoryStyle,
+  /** CC */ IdleForgetSkippable: memoryStyle,
+  /** CC */ ForgetSkippable: memoryStyle,
```

Comment:
I wonder if CC markers shouldn't be separated out. Or at least have different colors. I know that they're handled by completely different teams at Mozilla so that would make it easier for them.

Reply:
Let's follow-up with this. I keep on getting vague answers and understanding about the cycle collector and its relationship to the rest of it. I tried to get a good profile showing cycle collection, but everything is super short. I don't think this is actionable enough to file an issue with yet.

Reply:
As a reference, here is a profile with big GC and CC chunks: http://bit.ly/2GZiv4S